packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -14,7 +14,7 @@ import 'term.dart';
 enum _SymbolType { invalid, external, slack, error, dummy, }
 
 class _Symbol {
-  const _Symbol(this.type);
```

Comment:
I think that's accurate.  If you const-construct a symbol, you'll get strange aliasing behavior.  /cc @chinmaygarde